src/App.tsx

```diff
@@ -1,24 +1,66 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import { useEffect } from 'react';
+import { useTodoContext } from './context/TodoContext';
+import { getTodos } from './api/todos';
+import { Error } from './types/Error';
+import { TodoList } from './components/TodoList';
+import { TodoForm } from './components/TodoForm';
+import { FooterFilter } from './components/FooterFilter';
+import { ErrorMessage } from './components/ErrorMessage';
 
-const USER_ID = 0;
+const USER_ID = 10407;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const {
+    todos,
+    error,
+    setTodos,
+    setError,
+  } = useTodoContext();
+
+  const loadTodos = async () => {
+    try {
+      setError(null);
+      const todosFromServer = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    } catch {
+      setError(Error.LOAD);
```

Comment:
Replace 'catch' with 'catch (error)' to handle the error and provide a more specific message or log the error for debugging purposes.